src/main/java/net/sf/jabref/gui/fieldeditors/JTextAreaWithHighlighting.java

```diff
@@ -35,13 +34,15 @@
     private UndoManager undo;
 
     public JTextAreaWithHighlighting() {
-        super();
-        setupUndoRedo();
-        setupPasteListener();
+        this(null);
+    }
+
+    public JTextAreaWithHighlighting(String text) {
+        this(text, "");
     }
 
-    JTextAreaWithHighlighting(String text) {
-        super(text);
+    public JTextAreaWithHighlighting(String text, String title) {
```

Comment:
Please add javadoc comments at least describing `title`. Isn't title the `textWhenNotFocused`

Please rename `textWhenNotFocused` to `placeholder` to be in line with HTML5 --> http://www.w3schools.com/tags/att_input_placeholder.asp